frontend/test/__support__/e2e/config.js

```diff
@@ -61,6 +62,22 @@ const defaultConfig = {
       return launchOptions;
     });
 
+    /********************************************************************
+     **                           TASKS                                **
+     ********************************************************************/
+
+    on("task", {
+      async connectAndQueryDB({ connectionString, connectionConfig, query }) {
+        const client = new Client(connectionString || connectionConfig);
```

Comment:
There could be a pitfall here, if people send both `connectionString` and `connectionConfig`. Since we'll just use the string and ditch the config.

What do you think about adding an if check we need **exactly one** of the string of config to be present, otherwise, it won't work.

Maybe I'm just being paranoid, but I'd better be safe than sorry.

Reply:
I'm not sure why do we need both? Should we just stick to a single variant? As an idea, we can use a single parameter to accept both string/config variants like:


```js
connectAndQueryDB({ connection: `psql://host:port/db` })
connectAndQueryDB({ connection: { host: "...", port: 5432 } })
```